src/assets/thirdPartyServices/snet/yolov3_object_detection/index.js

```diff
@@ -4,40 +4,50 @@ import Button from "@material-ui/core/Button";
 import Slider from "@material-ui/lab/Slider";
 
 import SNETImageUpload from "../../standardComponents/SNETImageUpload";
+import { Detector } from "./object_detection_pb_service";
+
+
+const initialUserInput = {
+  methodName: "Select a method",
+  img_path: undefined,
+  confidence: 0.5,
+  img_path: "",
+  methodName: "detect",
```

Comment:
Both `image_path` and `methodName` have duplicated. Line `11` and `12` can be deleted.

Reply:
Done.